hazelcast/src/test/java/com/hazelcast/internal/usercodedeployment/impl/ClassDataProviderTest.java

```diff
@@ -82,6 +82,8 @@ private ClassDataProvider createClassDataProvider(UserCodeDeploymentConfig.Provi
     }
 
     private static ClassSource newMockClassSource() {
-        return new ClassSource(null, null);
+        ClassSource classSource = new ClassSource(null, null);
+        classSource.addClassDefinition("className", new byte[4]);
```

Comment:
newly added check broke `ivenProviderModeSetToLOCAL_AND_CACHED_CLASSES_whenMapClassContainsClass_thenReturnIt` which assumed ClassDataProvider would return an empty ClassData even when the requested className could not be found. This line is to fix the test.